recipes-extended/images/docker-image-minimal.bb

```diff
@@ -0,0 +1,7 @@
+require recipes-core/images/core-image-minimal.bb
+
+IMAGE_INSTALL_append = " \
+  docker-ce \
+  less \
+  openssh \
```

Comment:
@Asiderr I guess that you want to enable openssh server? You should use IMAGE_FEATURES_append = " ssh-server-openssh" instead. https://www.yoctoproject.org/docs/latest/ref-manual/ref-manual.html#ref-features-image